### PR TITLE
Fix wakeup agent. closes #2809

### LINKED ISSUE #2809: Task is "Prepared", but agents not waking up to execute it
**Describe the bug**

Task is "Prepared", but agents not waking up to execute it. Agents installed on Windows 7. Firewall is opened.

**Page(s) URL**

plugins/fusioninventory/front/task.form.php?id="id number for task"

**To reproduce**

1. Go to Deploy -> Package management
2. Create a package (For example - Action:Create directory on desktop)
3. Go to Tasks -> Task management
4. Create new task
5. Name it, and click add
6. Then, set task to active
7. Select or create timeslots for "Preparation" and "Execution". Timeslot is 00:00 - 24:00 for every day of week
8. Set "Agent wakeup interval (in minutes)" to 1
9. Set "Number of agents to wake up" to 1
10. Save configuration
11. Go to "Job configuration"
12. Create job with package created in step 2 and select computer running Windows
13. Go to "Task management" on left sidebar
14. Click "Force start"
15. Go to "Job executions" and see that task is "Prepared" and "Not done yet"
16. Wait 10 minutes and task is still "Prepared" and "Not done yet"

Agent will execute task after connection to server using agents schedule.

**Expected behavior**

1)Server sends signal to wake up agents
2)Agents execute task.

**Screenshots**

img_101
img_102
img_103

**GLPI setup**

```
Information about system installation and configuration

GLPI 9.4.2 ( => /var/www/glpi)
Installation mode: TARBALL

Server

Operating system: Linux glpi 4.9.0-7-amd64 #1 SMP Debian 4.9.110-3+deb9u2 (2018-08-13) x86_64
PHP 7.0.33-0+deb9u3 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, calendar, ctype,
	curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, json, ldap, libxml, mbstring, mysqli, mysqlnd,
	openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets, standard, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xml,
	xmlreader, xmlrpc, xmlwriter, xsl, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.25 (Debian) (Apache/2.4.25 (Debian) Server at glpi.energycollege.ru Port 443)
	Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0
Server Software: Debian 9.6
	Server Version: 10.1.37-MariaDB-0+deb9u1
	Server SQL Mode: 
	Parameters: db-glpi@localhost/glpi
	Host info: Localhost via UNIX socket

mysqli extension is installed
ctype extension is installed
fileinfo extension is installed
json extension is installed
mbstring extension is installed
iconv extension is installed
zlib extension is installed
curl extension is installed
gd extension is installed
simplexml extension is installed
xml extension is installed
ldap extension is installed
imap extension is installed
Zend OPcache extension is installed
xmlrpc extension is installed
exif extension is installed
APCu extension is not present
CAS extension is not present
Database version seems correct (10.1.37) - Perfect!Database version seems correct (10.1.37) - Perfect!
OK/var/www/glpidata/files : OK
OK/var/www/glpidata/config : OK
OK/var/www/glpidata/files/_dumps : OK
OK/var/www/glpidata/files/_sessions : OK
OK/var/www/glpidata/files/_cron : OK
OK/var/www/glpidata/files/_graphs : OK
OK/var/www/glpidata/files/_lock : OK
OK/var/www/glpidata/files/_plugins : OK
OK/var/www/glpidata/files/_tmp : OK
OK/var/www/glpidata/files/_cache : OK
OK/var/www/glpidata/files/_rss : OK
OK/var/www/glpidata/files/_uploads : OK
OK/var/www/glpidata/files/_pictures : OK
Web access to files directory is protectedWeb access to files directory is protected : OK

Libraries

htmLawed version 1.2.4 in (/var/www/glpi/lib/htmlawed)
phpmailer/phpmailer version 6.0.7 in (/var/www/glpi/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.2 in (/var/www/glpi/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.2.26 in (/var/www/glpi/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/glpi/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/glpi/vendor/iamcal/lib_autolink)
sabre/vobject in (/var/www/glpi/vendor/sabre/vobject/lib)
zendframework/zend-cache in (/var/www/glpi/vendor/zendframework/zend-cache/src)
zendframework/zend-i18n in (/var/www/glpi/vendor/zendframework/zend-i18n/src)
zendframework/zend-serializer in (/var/www/glpi/vendor/zendframework/zend-serializer/src)
monolog/monolog in (/var/www/glpi/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/glpi/vendor/sebastian/diff/src)
elvanto/litemoji in (/var/www/glpi/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/glpi/vendor/symfony/console)
leafo/scssphp in (/var/www/glpi/vendor/leafo/scssphp/src)

SQL replicas

Not active

Notifications

Way of sending emails: PHP

Plugins list

fusioninventory      Name: FusionInventory                Version: 9.4+1.1    State: Enabled
```

**Additional context**

Similar issue #2781

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

File: inc/agent.class.php [hunk 1]
```diff
@@ -1095,6 +1095,7 @@ public function getAgentRunURLs() {
       $ret = [];
 
       foreach ($this->getAgentBaseURLs() as $url) {
+         array_push($ret, $url."/now");
          array_push($ret, $url."/now/".$this->fields['token']);
```
Comment (g-bougard) [inline inc/agent.class.php]: Just a point, since b9cf48ab, adding token is no more needed on the agent side and this is just ignored.

Comment (ddurieux) [reply inc/agent.class.php]: I had tested yesterday and with the token seems not works, that's why I added the /now without the token

Comment (g-bougard) [reply inc/agent.class.php]: My purpose was to tell the line 1099 could be removed.
If you have a failure on that URL with the token, you still should check the agent log to see if we may have an error on agent side or if your problem would be elsewhere. From my point of view, the 2 URLs are valid for the agent so adding the URL is probably not the right fix.

Comment (ddurieux) [reply inc/agent.class.php]: You're right, I fix it

**Pull request merged**